### PR TITLE
Add Ray Train training to GitHub actions CI/CD test

### PR DESCRIPTION
**Pull request by erwulff**
- Add Ray Train training to GitHub actions CI/CD test 
- Save training stats to file after every epoch
- Fix dropout parameter in pyg-clic-hits config
- Fix Ray Train multi-gpu  dataloader error

### CODE REVIEW

**Comment by jpata** [comment]
Cool, I can see that the ray test now ran in the job, thanks a lot!
